python/mxnet/gluon/data/text/utils.py

```diff
@@ -75,3 +80,58 @@ def collate_pad_length(num_items, seq_len, overlap=0):
     step = seq_len-overlap
     span = num_items-seq_len
     return (span // step + 1) * step - span
+
+_vocab_sha1 = {}
+
+def _load_pretrained_vocab(name, root=os.path.join('~', '.mxnet', 'models')):
```

Comment:
I think using `~` here will not make it good for windows. We can consider to put this `root` variable to the _constants.py and set it based on the os.

Reply:
`~` is recognized by `os.path.expanduser` and works fine. vision model zoo already uses it.